ansible/roles/ssc/templates/ssc-deploy.yml.j2

```diff
@@ -0,0 +1,131 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ssc-data
+  namespace: {{ ssc_namespace }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ ssc.volume_size }}
+{% if ssc_storage_class %}
+  storageClassName: {{ ssc_storage_class }}
+{% endif %}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ssc
+  namespace: {{ ssc_namespace }}
+  labels:
+    app: ssc
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ssc
+  template:
+    metadata:
+      labels:
+        app: ssc
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "{{ ssc_metrics_port }}"
+    spec:
+      initContainers:
+        - name: init-data
```

Comment:
There is no need for this if you are then initializing with the second container below. So, pick one of the two approaches.